doc/tutorials/your_first_project/your_first_project.rst

```diff
@@ -0,0 +1,257 @@
+
+.. code-block:: C++
+
+  // Create the MoveIt MoveGroup Interface
+  using moveit::planning_interface::MoveGroupInterface;
+  auto move_group_interface = MoveGroupInterface(node, "panda_arm");
+
+  // Set a target Pose
+  auto const target_pose = []{
+    geometry_msgs::msg::Pose msg;
+    msg.orientation.w = 1.0;
+    msg.position.x = 0.28;
+    msg.position.y = -0.2;
+    msg.position.z = 0.5;
+    return msg;
+  }();
+  move_group_interface.setPoseTarget(target_pose);
+
+  // Create a plan to that target pose
+  auto const [success, plan] = [&move_group_interface]{
+    moveit::planning_interface::MoveGroupInterface::Plan msg;
+    auto const ok = static_cast<bool>(move_group_interface.plan(msg));
+    return std::make_pair(ok, msg);
+  }();
+
+  // Execute the plan
+  if(success) {
+    move_group_interface.execute(plan);
+  } else {
+    RCLCPP_ERROR(logger, "Planing failed!");
+  }
+
+3.1 Build and Run
+~~~~~~~~~~~~~~~~~
+
+Just like before, we need to build the code before we can run it.
+
+In the workspace directory ``ws_moveit2`` run this command:
+
+.. code-block:: bash
+
+  colcon build --mixin debug
+
+After this succeeds, we need to re-use the demo launch file from the previous tutorial to start RViz and the MoveGroup node.
+In a separate terminal, source the workspace and then execute this:
+
+.. code-block:: bash
+
+  ros2 launch moveit2_tutorials demo.launch.py
+
+Then in the ``Displays`` window under ``MotionPlanning/Planning Request`` uncheck the box ``Query Goal State``.
+
+.. image:: rviz_1.png
+   :width: 300px
+
+In a third terminal source the workspace and run your program.
+
+.. code-block:: bash
+
+  ros2 run hello_moveit hello_moveit
+
+This should cause the robot in RViz to move and end up in this pose:
+
+.. image:: rviz_2.png
+   :width: 300px
+
+Note that if you ran the node ``hello_moveit`` without launching the demo launch file first it will wait for 10 seconds and then print this error and exit.
+
+.. code-block:: bash
+
+  [ERROR] [1644181704.350825487] [hello_moveit]: Could not find parameter robot_description and did not receive robot_description via std_msgs::msg::String subscription within 10.000000 seconds.
+
+This is because the ``demo.launch.py`` launch is starting the ``MoveGroup`` node that provides the robot description.
+When ``MoveGroupInterface`` is constructed it looks for a node publishing a topic with the robot description.
+If it fails to find that within 10 seconds it prints this error and terminates the program.
+
+3.2 Examine the code
+~~~~~~~~~~~~~~~~~~~~
+
+The first thing we do is create the MoveGroupInterface. This object will be used to interact with move_group, which allows us to plan and execute trajectories.
+Note that this is the only mutable object that we create in this program.
+
+.. code-block:: C++
+
+  using moveit::planning_interface::MoveGroupInterface;
+  auto move_group_interface = MoveGroupInterface(node, "panda_arm");
```

Comment:
maybe some explanation above about what the "panda_arm" argument is for?

Reply:
I have updated this based on your feedback, would you mind looking over my changes and letting me know if you think they are what you were hoping for?

Reply:
lgtm!